src/data/fin.lean

```diff
@@ -304,6 +304,12 @@ by { cases i, refl }
   fin.pred ⟨i + 1, add_lt_add_right h 1⟩ (ne_of_vne (ne_of_gt (mk_succ_pos i h))) = ⟨i, h⟩ :=
 by simp only [ext_iff, coe_pred, coe_mk, nat.add_sub_cancel]
 
+@[simp] lemma pred_le_pred_iff {n : ℕ} {a b : fin n.succ} {ha hb} : a.pred ha ≤ b.pred hb ↔ a ≤ b :=
```

Comment:
Do you mind spelling out the types for `ha` and `hb` here and below?

Otherwise LGTM.
bors d+